sunkit_pyvista/plotter.py

```diff
@@ -377,3 +383,27 @@ def load(self, filepath):
         file_path = Path(filepath)
         mesh_block = pv.read(file_path)
         self._loop_through_meshes(mesh_block)
+
+    def plot_limb(self, m, radius=0.02, **kwargs):
+        """
+        Draws the solar limb as seen by the map's observer.
+
+        Parameters
+        ----------
+        m : `sunpy.map.Map`
+            Map's limb to be plotted.
+        radius : `float`
+            Radius of the `pyvista.Spline` used to create the limb.
+            Defaults to ``0.02`` times the radius of the sun.
+        **kwargs : Keyword arguments are handed to `pyvista.Plotter.add_mesh`.
+        """
+        limb_coordinates = get_limb_coordinates(m.observer_coordinate, m.rsun_meters,
+                                                resolution=1000)
```

Comment:
We should allow the user to specify the resolution as a keyword argument to this method.

Reply:
Ah, we're using Spline here so I don't think resolution matters right?

Reply:
I guess that's true